src/test/java/org/junitpioneer/jupiter/report/PioneerReportListenerTest.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright 2015-2020 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junitpioneer.jupiter.report;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Collections;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.parallel.Execution;
+import org.junit.jupiter.api.parallel.ExecutionMode;
+import org.junit.platform.engine.TestDescriptor;
+import org.junit.platform.engine.TestExecutionResult;
+import org.junit.platform.engine.UniqueId;
+import org.junit.platform.engine.reporting.ReportEntry;
+import org.junit.platform.engine.support.descriptor.AbstractTestDescriptor;
+import org.junit.platform.launcher.TestIdentifier;
+import org.junit.platform.launcher.TestPlan;
+
+@Execution(ExecutionMode.SAME_THREAD)
+public class PioneerReportListenerTest {
+
+	private final PioneerReportListener listener = new PioneerReportListener();
+	private final TestPlan testPlan = TestPlan.from(Collections.emptyList());
+
+	@Test
+	void emptyReport() {
+		listener.testPlanExecutionStarted(testPlan);
+		listener.testPlanExecutionFinished(testPlan);
+
+		// Verify result
+		PioneerReport report = listener.getPioneerReport();
+		assertThat(report.getProperties()).isNull();
+		assertThat(report.getTestcontainer().isEmpty()).isTrue();
+		assertThat(report.getTestcase().isEmpty()).isTrue();
+	}
+
+	@Test
+	void reportingCorrectCounts() {
+
+		ReportEntry succContainerEntry = ReportEntry.from("key: succContainer", "value: succContainer");
```

Comment:
(Are you aware that there succ is a slang term? Just saying...)

Reply:
I'm not, but I always write _successful_ wrong (mixing single s and double f instead of correct double s and single f).

Renamed